trino/client.py

```diff
@@ -912,6 +919,7 @@ def round_to(self, precision: int) -> TemporalType:
             In case the supplied value exceeds the specified precision,
             the value needs to be rounded.
         """
+        precision = min(precision, 6)
```

Comment:
Please create a constant with a descriptive name for the max python temporal precision.

Reply:
Done